packages/grafana-ui/src/components/ThresholdsEditor/ThresholdsEditor.tsx

```diff
@@ -1,14 +1,19 @@
 import React, { PureComponent, ChangeEvent } from 'react';
-import { Threshold } from '../../types';
+
 import { ColorPicker } from '..';
 import { PanelOptionsGroup } from '..';
+
 import { colors } from '../../utils';
-import { ThemeContext } from '../../themes/ThemeContext';
-import { getColorFromHexRgbOrName } from '../../utils/namedColorsPalette';
+import { getColorFromHexRgbOrName } from '../../utils';
+
+import { ThemeContext } from '../../themes';
+
+import { Threshold } from '../../types';
 
 export interface Props {
   thresholds: Threshold[];
   onChange: (thresholds: Threshold[]) => void;
+  withText?: boolean;
```

Comment:
@peterholmberg WDYT about naming this `enableThresholdText` | `enableThresholdLabels`?  

Since this is editor we want to enable certain functionalities, while `withText` suggest, that something will be provided to the component.

Reply:
Also, I suggest adding jsdoc comment for this prop to describe what's it for.

Reply:
Yeah, I see what you mean with "enabling" instead of "with". Will update that.